xwiki-commons-core/xwiki-commons-xml/src/main/java/org/xwiki/xml/internal/html/DefaultHTMLCleaner.java

```diff
@@ -221,6 +221,9 @@ private CleanerProperties getDefaultCleanerProperties(HTMLCleanerConfiguration c
         // TODO: handle HTML5 correctly (see: https://jira.xwiki.org/browse/XCOMMONS-901)
         defaultProperties.setHtmlVersion(4);
 
+        // Don't trim spaces in attribute values
```

Comment:
Could you explain why in the comment, maybe by giving an example? I guess that trimming is the default so if we go against the default we should explain why so that we remember it in 2 years time ;) Thx

Reply:
I can add in the comment my test example, if it's what you have in mind.

Reply:
What I have in mind is to document why we need to trim spaces. Thx